dds_web/static/swagger.yaml

```diff
@@ -162,11 +184,12 @@ paths:
     delete:
       tags:
         - files
-      summary: Deletes a file CHECK METHOD
+      summary: Deletes a file metadataCHECK METHOD
       description: This method requires the data
         to be passed in the request body instead of the query.
         Since this does not comply with the openAPI standards, swagger cannot document it properly,
-        therefore we need to change/remove it in the future.
+        therefore we need to change/remove it in the future. <br><br>
+        Deletes the file from the DB and bucket, no further operation with the s3 keys are required.
```

Comment:
Maybe it can be better explained. It was because there is another endpoint for the s3 keys. 
And, for example, the endpoint to add a file just adds the metadata, whereas this also tries to delete the file from the bucket. So I was a bit confused at first